sympy/integrals/tests/test_integrals.py

```diff
@@ -20,6 +21,31 @@
 n = Symbol('n', integer=True)
 f = Function('f')
 
+def test_principal_value():
+
```

Comment:
There should also be tests for cases like `Integral(x).principal_value()` and `Integral(x, x).principal_value()`.

Reply:
It should be checked that there are three values before trying to unpack them.